great_expectations/core/usage_statistics/anonymizers/anonymizer.py

```diff
@@ -190,3 +193,290 @@ def _get_parent_class(
             pass
 
         return None
+
+    def anonymize_action_info(self, action_name, action_obj=None, action_config=None):
```

Comment:
Type hints and doc strings to come in a subsequent PR - this is simply porting over all behavior from child classes to this primary class.

The removal of `self._ge_classes` removes the need for most of the child classes (since the only thing differentiating them was the aforementioned list).